packages/angular-mercury/src/app/app.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { RouterOutlet } from "@angular/router";
+import { CustomDialogComponent } from "./custom-dialog/custom-dialog";
+
+import { ThemeModel } from "@genexus/chameleon-controls-library";
+import { bundleToHashMappings } from "../assets/mercury-bundles/bundle-to-hash-mappings";
+@Component({
+  selector: "app-root",
+  standalone: true,
+  imports: [RouterOutlet, CustomDialogComponent],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  templateUrl: "./app.html",
```

Comment:
Angular components must use `OnPush` change detection strategy for better performance.

You can see an example here (selected the "Angular" tab in a code block):
https://mercury-showcase.genexus.com/mercury/components/button

```suggestion
  changeDetection: ChangeDetectionStrategy.OnPush,
  templateUrl: "./app.html",
```